package.json

```diff
@@ -46,11 +47,11 @@
       "https://mail.google.com/*",
       "https://docs.google.com/*",
       "https://encrypted.google.com/*",
-      "https://mastodon.social/*",
-      "http://mxr.mozilla.org/*",
+      "https://*.101weiqi.com/*",
```

Comment:
This won't match `https://101weiqi.com/guan/`. Note that you are modifying an `@exclude` rule, which is a [glob pattern](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/content_scripts#matching_url_patterns).

In order to match both `https://101weiqi.com/guan/` and `https://www.101weiqi.com/q/14910/`, you have to use something like:
```
https://*101weiqi.com/*
```
Or, just add two hosts:
```
https://101weiqi.com/*
https://www.101weiqi.com/*
```